.github/pull_request_template.md

```diff
@@ -1,5 +1,7 @@
 ## Requester Checklist
 
+Please only check the items that you have done. Do not check items that are not applicable to your PR.
```

Comment:
How far do we want to take this? E.g `Have all OneBlink dependency updates been completed (eg apps / apps-react / types etc)` => if 0 dependencies need updating, then doing 0 of them is a tick?

Reply:
0 dependencies === not applicable IMO

Anyone else have an opinion?

Reply:
Another example:

if your PR contains no numbers (hard coded or in variables), their is no need to tick:

> There is no usage of [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))

Reply:
Examples sound good. Potentially could highlight the subtle difference with something this as well: 
```suggestion
Please only check the items that you have actioned. Do not check items that are not applicable to your PR.
```

Reply:
The ideal way to handle some of these scenarios (in my opinion), would be to have checkboxes and sub checkboxes. Eg.

- Does this PR require OB dependency updates (eg apps / apps-react / types etc)?
  - Have you updated all OB dependencies?

- Does this PR include changes that will knowingly make the feature/bug incomplete?
  - Have they been commented with TODO and a description of what needs to be done to finish the feature/bug?

Reply:
If that's possible in MD, I assume it probably is.

Reply:
Also, I don't expect you to do this right now, but it does seem like it should be the ultimate goal to me.